Update export/import and migration docs to include CaC projects (#2661)
OctopusDeploy/docs: src/pages/docs/projects/export-import/index.mdx

To find out more take a look at our examples:
- [Export projects](/docs/octopus-rest-api/examples/projects/export-projects)
- [Import projects](/docs/octopus-rest-api/examples/projects/import-projects)

## Older versions

> - Prior to version **2025.2.5601**, version-controlled projects were not supported by the **Export/Import Projects** feature.

Reviewer: I am no expert at English, it feels like this could have the logic reversed:

```suggestion
- Importing and Exporting of version-controlled projects is only supported from version **2025.2.5601**
```

Author: I agree that it feels backward when you read it, but it's the logical process you go through though. And we use that order elsewhere in these docs.

Reviewer: If it is used elsewhere then leave it as is
